Example/AblyChatExample/Mocks/MockClients.swift

```diff
@@ -1,6 +1,7 @@
 import Ably
 @testable import AblyChat
 
+@MainActor
```

Comment:
Why have you added this to all of the mocks? The protocols they implement are already `@MainActor`.

Reply:
removed